core/src/luigi-config/navigation/navigation-data-init.js

```diff
@@ -77,7 +77,10 @@ export function getNavigationData(authData) {
             clearAuthData();
             window.location.pathname = '/nopermissions.html';
           } else {
-            alert('Config init error, see console for more details');
+            Luigi.ux().showAlert({
+              text: `Could not load initial configuration: ${err.message}`,
+              type: 'error',
+            });
             console.warn(err);
```

Comment:
```suggestion
            console.warn(err.message);
```

Reply:
I don't think we need this in `console` calls:
![image](https://user-images.githubusercontent.com/10504661/114183928-800b9280-9944-11eb-88f5-29e391b55fc0.png)
When we show only message, we don't get the call stack.